wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -0,0 +1,56 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Subheader
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader {
+	position: relative;
+	display: flex;
+	align-items: center;
+	min-height: 260px;
+	background: var(--color-grey-light);
+	padding:  var(--spacer-20) 0;
+	color: var(--color-text);
+	margin-bottom: var(--spacer-80);
+}
+
+/* CASE: Page with featured image */
+body:not(.page-page-with-no-featured-image) .c-subheader { 
+	min-height: 440px;
+	align-items: flex-end;
+	padding:  var(--spacer-20) 0 var(--spacer-80) 0; 
+	color: var(--color-white);		
+}
```

Comment:
A few thoughts on this:
1. We generally prefer to see classes added in the positive, in other words, `page-with-featured-image`, when the subheader has a featured image instead of adding a class when the image is missing.
2. Please use a BEM-style modifier such as `page--has-featured-image`.
3. Imho, this css class should be added to the c-subheader component classes. Something like: `c-subheader--has-image`, rather than making the css for this component dependent on a body class, which is outside the component. 
4. It's preferable to ad this as a modifier within the `c-subheader` declaration above, rather than a separate declaration. Something like:

```
.c-subheader {
	position: relative;
	display: flex;
	align-items: center;
	min-height: 260px;
	padding:  var(--spacer-20) 0;
	margin-bottom: var(--spacer-80);
	
	/* CASE: Subheader has image */
	&.c-subheader-has-image {
		min-height: 440px;
	}
}
```
This keeps related styles together and avoid using a high-priority selector (`:not()`).

Reply:
Thanks, I will work on this!